artifacts/source_type.py

```diff
@@ -426,14 +426,23 @@ def DeregisterSourceType(cls, source_type_class):
 
   @classmethod
   def GetSourceTypes(cls):
-    """Retriees the source types.
+    """Retrieves the source types.
 
     Returns:
       A list of source types (subclasses of SourceType).
     """
     return cls._source_type_classes.values()
 
   @classmethod
+  def GetSourceTypeIndicators(cls):
```

Comment:
They are referred to as TypeIndicators everywhere else.  I'm not committed to either name, but I want to be sure everyone wants the rename because I have to change other code of mine any time this method changes.